core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -438,7 +438,7 @@ public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throw
    *
    * @return true if zookeeper should be used
    */
-  protected abstract boolean isZookeeperMode();
+  protected abstract boolean isZooKeeperMode();
```

Comment:
Would there be any issue with backwards compatibility of older clients with the old name?